wordpress-mobile/gutenberg-mobile
# Focus on RichText when selection changes

This combines https://github.com/WordPress/gutenberg/pull/12422 and https://github.com/wordpress-mobile/react-native-aztec/pull/85

When a RichText becomes selected, it will call `focus` on the aztec view to ensure it gets the right focus. The focus was working OK when tapping, but this fixes the cases where the block focus changes programatically, like inserting a new paragraph or splitting a block.

To test:

- Insert a new paragraph, verify it has focus
- Type some words, press return, verify the new block has focus
